src/components/scenes/PluginViewScene.js

```diff
@@ -252,6 +324,7 @@ class PluginView extends React.Component<PluginProps, PluginState> {
           return
         }
         this.openingSendConfirmation = true
+        console.log('a1: parsed uri ', data['edge-uri'])
```

Comment:
Extraneous debug logging. Please remove for production

Reply:
fixed